README.md

```diff
@@ -30,6 +30,14 @@ Commands useful for migrations
 Commands useful in Conan Center Index or its forks
 
 
+### Deployer
+
+These are the currently supported custom commands:
```

Comment:
```suggestion
These are the currently supported custom deployers:
```

But we're now creating a README per folder, so one in extensions/deploy would be nice. The file won't be copied to the user's cache